### PR TITLE
transform paramref, typeparamref and languageKeyword to <code>

### PR DESCRIPTION
**Pull request by TianqiZhang**

### CODE REVIEW

File: src/Microsoft.DocAsCode.Metadata.ManagedReference/Transform/TripleSlashCommentTransform.xsl [hunk 1]
```diff
@@ -98,20 +92,26 @@
 
   <xsl:template match="paramref">
     <xsl:if test="normalize-space(@name)">
-      <span class="paramref">
+      <code data-dev-comment-type="paramref">
```
Comment (hellosnow) [inline src/Microsoft.DocAsCode.Metadata.ManagedReference/Transform/TripleSlashCommentTransform.xsl]: @vicancy  we'd better transfer the element into span with specific class in our javascript?

**Comment by superyyrrzz** [comment]
@TianqiZhang  Unit test fails. Please update test cases.

**Comment by vicancy** [comment]
Update https://github.com/dotnet/docfx/blob/dev/src/docfx.website.themes/default/styles/docfx.css#L88 accordingly

File: src/Microsoft.DocAsCode.Metadata.ManagedReference/Transform/TripleSlashCommentTransform.xsl [hunk 2]
```diff
@@ -98,20 +92,26 @@
...
         <xsl:value-of select="@name" />
-      </span>
+      </code>
     </xsl:if>
   </xsl:template>
 
   <xsl:template match="typeparamref">
     <xsl:if test="normalize-space(@name)">
-      <span class="typeparamref">
+      <code data-dev-comment-type="typeparamref">
```
Comment (vicancy) [inline src/Microsoft.DocAsCode.Metadata.ManagedReference/Transform/TripleSlashCommentTransform.xsl]: naming: why it is called "dev comment" type `data-dev-comment-type`

File: src/Microsoft.DocAsCode.Metadata.ManagedReference/Transform/TripleSlashCommentTransform.xsl (hunk 1 above)
Comment (vicancy) [inline src/Microsoft.DocAsCode.Metadata.ManagedReference/Transform/TripleSlashCommentTransform.xsl]: @hellosnow  I think update default.css to support the change is enough

**Comment by hellosnow** [comment]
:shipit:

**Pull request merged**